go.mod

```diff
@@ -2,11 +2,14 @@ module github.com/osbuild/image-builder-cli
 
 // keep in sync with images,
 // c.f. https://github.com/osbuild/images/blob/main/go.mod
-go 1.21.0
+go 1.22.0
```

Comment:
This is a bit of an odd one, bib is at 1.22 while images is at 1.21

Reply:
Well; I guess it should stick at the lowest common denominator being 1.21 right?

Reply:
Didn't we agree on the highest supported by all distros to be 1.22?

Reply:
(i.e. all of our supported distros ;) )

Reply:
I don't know, if we did then it should be 1.22 in images 😄.

Reply:
agree - I'm not the right person to verify that 1.22 is supported "everywhere", just vaguely remember this is the way :-D

Reply:
1.22 is in RHEL 9.5 so we're good to go for that.
We can move to 1.23 in images as soon as the IB service container (the UBI) starts using 9.6.

Reply:
Cool, great to hear that this is not a blocker.